### ISSUE TITLE
Total Holdings is summing Vault and Strategy Holdings

### ISSUE THREAD

**Issue by dudesahn** [iearn-finance/iearn-finance#175]
On the yearn.finance/stats page, Total Holdings is summing Vault and Strategy Holdings, which is a misleading number (and not a useful metric), as Vault Holdings already includes everything in the Strategy. 

<img width="1647" alt="Screen Shot 2020-10-28 at 3 38 29 PM" src="https://user-images.githubusercontent.com/23222916/97488083-25512200-1934-11eb-82b5-a2be0bbccc9a.png">

For now, I would advocate for simply removing the "Total Holdings" box and only having three (centered) boxes at the top of the stats page.

**Comment by WrongNebula** [comment]
As an end user, Total holdings should be the TVL, it should be the most accurate TVL (compared to stats.finance and DeFi pulse or debanking)

**Comment by dudesahn** [comment]
@WrongNebula I agree totally. I would argue for including the v2/v3 yEarn contracts as well in "Total Holdings". 

However, TVL in this case (if we're just looking at yVaults, which seems to be what this page is focused on) is the same thing as "Total Vault Holdings". If we add in the holdings in all of the yEarn contracts, then I think that would be a good "Total Holdings".
